fashionStart.m

```diff
@@ -281,6 +288,31 @@
     if r == true
         c = (numCorrect / (t * batchSize) * 100);
     end 
+
+        currentIteration = toc(iterationTime);
+        
+        %Convert from seconds to milliseconds
+        currentIteration = currentIteration * 1000;
+        
+%Add it to average time variable
+averageRunTime = averageRunTime + currentIteration;
+        
+    
+ 
+    
+%Average the total time by dividing by number of times loop runs
+averageRunTime = averageRunTime / t;
+
+
+
+disp("Average Run Time: ");
+fprintf('%d milliseconds\n',averageRunTime);
+    
+
+disp("Prediction: ");
```

Comment:
I like the idea of seeing the Predictions, but we might not want to print it out every time right now so we can more easily compare the correctness of the networks.

Reply:
We can change this in a future commit however.

Reply:
Agreed, I just had it there as an example of how to be used